### ISSUE TITLE
Star Rating Improvement

### ISSUE THREAD

**Issue by erich3000** [Khao-UI-Tools/khao-ui#139]
The Current Implementation of Star Rating ins very buggy. perhaps we should have a look a that a11y example: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/

**Comment by erich3000** [comment]
Imported to khao-ui/conponents/0003_star-rating-improvement.md
